### DISCUSSION TITLE
Filling missing values

### DISCUSSION THREAD

**Discussion by sami-ka** [feature-engine/feature_engine#656] [Q&A]
Hi, 

Thanks a lot for the tremendous work you put into this package!

I had to implement time series features on my own before discovering this package, and I have 2 questions about what is doable with Feature-engine:
* Let's say you want to compute the average number of purchase by day in the last 3 months out of a transaction dataset for example. There might be some days where there is no purchase, in that case you want to impute a 0 for any day with missing purchase. If I want to compute the average margin percentage each day, it will be different as I would like to leave the days without purchase and drop the NaNs (as it is probably currently done in the package). 
Is this imputation feasible right now? Or do you expect to have the whole set of dates given with the already filled nan values? Because this would imply an extra step compared to working directly with the transaction dataset.
* Is there a way to save the time series feature computation for all possible dates? To keep going with my previous example, I would like to keep the average number of purchase by day for every possible dates and store it somewhere. This would allow me to not recompute everything when I want feature information at a specific date that is not present in my transaction dataset.

Looking forward to your answers!

**Comment by solegalli** [comment]
Hi @drskd 

The transformer uses pandas.rolling together with pandas.agg under the hood. It does not do any extra manipulation (ie dropping na, filling with 0).

We are at first stages with time series, so we only offer very basic functionality.

If you have a specific request, please raise an issue with details of what you would like the transformer to do. If you have more than 1 request, please raise more than 1 issue ;)

Thank you!
